server/interceptor/cors/cors.go

```diff
@@ -29,41 +29,49 @@ func New() *CORS {
 	c := new(CORS)
 	c.allowOrigin = "*"
 	c.allowCredentials = false
-	c.allowHeaders = map[string]bool{"origin": true}
-	c.allowMethods = map[string]bool{"GET": true, "POST": true, "PUT": true, "DELETE": true, "UPDATE": true}
+	c.allowHeaders = map[string]struct{}{"origin": {}, "x-domain-name": {}, "x-consumerid": {}}
```

Comment:
Since All the header are case sensitive so we should put "X-ConsumerId" instead of "x-consumerid"